docs/src/main/mdoc/typeclasses/bifoldable.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: docs
+title:  "Bifoldable"
+section: "typeclasses"
+source: "core/src/main/scala/cats/Bifoldable.scala"
+scaladoc: "#cats.Bifoldable"
+---
+
+# Bifoldable
+
+`Bifoldable` takes two type parameters and it's seen as a `Foldable` in both of these parameters.
+
+As a reminder `Foldable` is implemented in terms of `foldLeft` and `foldRight`, similarly `Bifoldable` is implemented in terms of:
+
+   ```scala
+   //eagerly performs a left-associative bi-fold over `fab` 
+    def bifoldLeft[A, B, C](fab: F[A, B], c: C)(f: (C, A) => C, g: (C, B) => C): C
+
+    //lazily performs a right-associative bi-fold over `fab` 
+    def bifoldRight[A, B, C](fab: F[A, B], c: Eval[C])(f: (A, Eval[C]) => Eval[C], g: (B, Eval[C]) => Eval[C]): Eval[C]
+   ```
```

Comment:
The indentation seems a bit strange here. Also just my personal nit but `// space please` :)